CHEWBBACA/chewBBACA.py

```diff
@@ -1,254 +1,573 @@
 #!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+AUTHORS
+
+    Mickael Silva
+    github: @
+
+    Pedro Cerqueira
+    github: @pedrorvc
+
+    Rafael Mamede
+    github: @rfm-targa
+
+DESCRIPTION
+
+"""
+
 
-import sys
 import os
+import sys
+import pickle
+import shutil
+import datetime
 import platform
 import argparse
-from Bio import SeqIO
-from Bio.Alphabet import generic_dna
 
 try:
+    from __init__ import __version__
     from allelecall import BBACA
     from createschema import PPanGen
     from SchemaEvaluator import ValidateSchema
     from PrepExternalSchema import PrepExternalSchema
-    from utils import TestGenomeQuality,profile_joiner,init_schema_4_bbaca,uniprot_find,Extract_cgAlleles,RemoveGenes
-except ImportError:
+    from utils import (TestGenomeQuality, profile_joiner,
+                       uniprot_find, Extract_cgAlleles,
+                       RemoveGenes, sqlite_functions as sq,
+                       auxiliary_functions as aux,
+                       constants as cnts,
+                       parameters_validation as pv)
+
+    from utils.parameters_validation import ModifiedHelpFormatter
+
+    from CHEWBBACA_NS import (down_schema, load_schema,
+                              sync_schema, down_profiles,
+                              send2NS, send_metadata,
+                              stats_requests)
+except:
+    from CHEWBBACA import __version__
     from CHEWBBACA.allelecall import BBACA
     from CHEWBBACA.createschema import PPanGen
     from CHEWBBACA.SchemaEvaluator import ValidateSchema
     from CHEWBBACA.PrepExternalSchema import PrepExternalSchema
-    from CHEWBBACA.utils import TestGenomeQuality,profile_joiner,init_schema_4_bbaca,uniprot_find,Extract_cgAlleles,RemoveGenes
+    from CHEWBBACA.utils import (TestGenomeQuality, profile_joiner,
+                                 uniprot_find, Extract_cgAlleles,
+                                 RemoveGenes, sqlite_functions as sq,
+                                 auxiliary_functions as aux,
+                                 constants as cnts,
+                                 parameters_validation as pv)
 
-import CHEWBBACA
+    from CHEWBBACA.utils.parameters_validation import ModifiedHelpFormatter
 
+    from CHEWBBACA.CHEWBBACA_NS import (down_schema, load_schema,
+                                        sync_schema, down_profiles,
+                                        send2NS, send_metadata,
+                                        stats_requests)
 
```

Comment:
The following imports need to be removed because the files have been deleted: `down_profiles, send2NS, send_metadata`